WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -196,18 +196,18 @@ public void onFocusChange(View v, boolean hasFocus) {
             populateUsernameButtons(usernames);
         }
 
-        View roleContainer = view.findViewById(R.id.role);
-        roleContainer.setOnClickListener(new View.OnClickListener() {
+
+        mRoleTextView = (TextView) view.findViewById(R.id.role);
+        setRole(role);
```

Comment:
Although it doesn't change the runtime behaviour of the code in any meaningful way, I'd suggest moving the `setRole(role)` line back to where it was before. Reason is that now, having it sandwiched between the `findViewById` and the `setOnClickListener` makes it a bit more confusing for the code reviewer (like, "hmm, why is the setRole important to happen at that point?).

If this sounds like nitpicking is because it kinda is, but at the same time I think is useful to keep the code in familiar patterns. Thanks!